plugins/modules/helm_info.py

```diff
@@ -53,6 +53,14 @@
     type: list
     elements: str
     version_added: "2.3.0"
+  get_all_values:
+    description:
+      - Set to C(True) if you want to get all (computed) values of the release.
```

Comment:
```suggestion
      - Set to I(True) if you want to get all (computed) values of the release.
```

Reply:
`I()` is for option names and `C()` is for option values, so what is here is correct.

https://docs.ansible.com/ansible/latest/dev_guide/developing_modules_documenting.html#linking-and-other-format-macros-within-module-documentation